nilearn/_utils/glm.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+import pandas as pd
+
+from nilearn._utils import stringify_path
+
+
+def check_and_load_tables(tables_, var_name):
```

Comment:
moved to a new module as this private function is now also used outside of nilearn.glm